_testdata/releaseVersion.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+service CoreAPIContracts {
+    option (api.section) = "Contrats";
```

Comment:
Don't you have to import the "api.proot" before you can use it?

Reply:
I think catching that is in the scope of the linter rather than the parser?

Reply:
I'm not entirely familiar with how the parser works here, so I assumed it needs to know about all used symbols.